### PR TITLE
Branch Amenities: Closed Amenities

### PR DESCRIPTION
**Pull request by vitalii-stepaniuk**
This PR adds Closed Amenities field to Branch content type, displays amenities in two columns (Open and Closed amenities, if closed amenities exist), and adds a paragraph Branch Amenities that shows amenities with green check (open) and red cross (closed) icons.

## Steps for review

- [ ] Run updates.
- [ ] Edit any branch and add closed amenities inside Branch Amenities fieldset.
- [ ] Add Branch Amenities paragraph to the node.

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: modules/custom/openy_ckeditor/openy_ckeditor.module [hunk 1]
```diff
@@ -81,6 +76,31 @@ function openy_ckeditor_tokens($type, $tokens, array $data, array $options, Bubb
...
+ *
+ * @return array $amenities_view
+ */
+function _openy_ckeditor_prepare_amenities_view($field_value) {
+  $amenities_ids = [];
+  foreach ($field_value as $id) {
+    $amenities_ids[] = $id['target_id'];
+  }
+  $amenities = \Drupal::service('entity_type.manager')
+    ->getStorage('taxonomy_term')
```
Comment (anpolimus) [inline modules/custom/openy_ckeditor/openy_ckeditor.module]: TaxonomyTerm::loadMultiple

**Review by anpolimus — CHANGES_REQUESTED** [review]

File: modules/custom/openy_ckeditor/openy_ckeditor.module (hunk 1 above)
Comment (vitalii-stepaniuk) [reply modules/custom/openy_ckeditor/openy_ckeditor.module]: Fixed

**Comment by AndreyMaximov** [comment]
ok to test

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2206/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2206/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2206/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2206/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2206/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2206/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2206/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2206/

File: modules/openy_features/openy_block/modules/openy_block_branch_amenities/config/install/paragraphs.paragraphs_type.branch_amenities.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies: {  }
+_core:
+  default_config_hash: c4A5enOspH2lSRkObw2PzbZcD-ldXLpBO91Nk0F6Qas
```
Comment (podarok) [inline modules/openy_features/openy_block/modules/openy_block_branch_amenities/config/install/paragraphs.paragraphs_type.branch_amenities.yml]: it looks wrong. Hash should not be included in configs

File: modules/openy_features/openy_block/modules/openy_block_branch_amenities/config/install/field.storage.paragraph.field_amenities_block.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - block_content
+    - paragraphs
+_core:
+  default_config_hash: n4KePQPN8a6yMnGHTR-lYrOMBMOUQXhNQ89Q80fLjjg
```
Comment (podarok) [inline modules/openy_features/openy_block/modules/openy_block_branch_amenities/config/install/field.storage.paragraph.field_amenities_block.yml]: it looks wrong. Hash should not be included in configs

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_view_display.node.branch.default.yml [hunk 4]
```diff
@@ -26,11 +27,21 @@ dependencies:
     - openy_field_custom_hours
     - telephone
     - user
+_core:
+  default_config_hash: mRQJNhp4t6OSJSFQmOqvzFti_RolSTMMSKpRgcezXNE
```
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/core.entity_view_display.node.branch.default.yml]: it looks wrong. Hash should not be included in configs

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_amenities_closed.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+third_party_settings:
+  datalayer:
+    expose: 0
+    label: field_location_amenities_closed
+id: node.branch.field_location_amenities_closed
+field_name: field_location_amenities_closed
+entity_type: node
+bundle: branch
+label: 'Closed Amenities'
+description: ''
```
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_amenities_closed.yml]: can we have a business description everywhere?

File: modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.storage.node.field_location_amenities_closed.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+entity_type: node
+type: entity_reference
+settings:
+  target_type: taxonomy_term
+module: core
+locked: false
+cardinality: -1
+translatable: true
+indexes: {  }
+persist_with_no_fields: false
```
Comment (podarok) [inline modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.storage.node.field_location_amenities_closed.yml]: always true, please

File: themes/openy_themes/openy_carnation/src/scss/paragraphs/_amenities.scss [hunk 7]
```diff
@@ -216,3 +216,87 @@
...
+  font-weight: bold;
+}
+
+.paragraph--type--branch-amenities {
+  border: none;
+  background: #fff;
+  box-shadow: 0 3px 30px 2px rgba(0, 0, 0, 0.05);
+  border-radius: 8px;
+  margin-bottom: 50px;
+  color: #4f4f4f;
```
Comment (podarok) [inline themes/openy_themes/openy_carnation/src/scss/paragraphs/_amenities.scss]: can we use global _variables where all colors stored?

File: themes/openy_themes/openy_carnation/src/scss/paragraphs/_amenities.scss [hunk 8]
```diff
@@ -216,3 +216,87 @@
...
+    .term-name .name {
+      display: inline-block;
+      padding-left: 8px;
+    }
+
+    .amenities-closed .term-name:before {
+      font-family: $font-awesome;
+      font-weight: 900;
+      content: "\f00d";
+      color: #dc3545;
```
Comment (podarok) [inline themes/openy_themes/openy_carnation/src/scss/paragraphs/_amenities.scss]: same here, global colors, please

**Review by podarok — CHANGES_REQUESTED** [review]
Small changes, please

**Comment by vitalii-stepaniuk** [comment]
@podarok The issues you've listed above are fixed now. Could you please review the PR again?
Thanks!

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2207/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2207/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2207/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2207/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2207/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2207/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2207/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2207/

**Review by podarok — DISMISSED** [review]

**Comment by podarok** [comment]
@vitalii-stepaniuk fix conflicts and this one ready to be merged

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2212/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2212/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2212/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2212/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2212/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2212/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2212/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2212/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2213/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2213/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2213/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2213/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2213/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2213/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2213/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2213/

**Pull request merged**